### ISSUE TITLE
Setting eligibility to issue a BadgeClass that requires sets of badges earned

### ISSUE THREAD

**Issue by szerge** [openbadges/openbadges-specification#75]
**Problem**: Earned badges are statements of trust and authorization and should be possible to use to indicate privileges within badge systems. The profile of an issuer, and more importantly, badges and endorsements received by that profile, is critical to allow/forbid certain actions.

**Requirement**: A BadgeClass's creator (Issuer) may set requirements that other Issuers may meet in order to be viewed as an authorized issuer of the badge. For example in plain English, "issuers must have Badge XYZ22 and Badge Y204". This could be implemented by issuing platforms before allowing issuing either to authorize the user to act as the BadgeClass's creator or allowing them to create an Assertion under their own Issuer profile. Alternately, this could be enforced post-facto by verification tools, which would insist on receiving a record of whether the issuing conditions were met.

**Motivation**: Organisations need to delegate the right to issue badges on their behalf. They create BadgeClasses defining the issuing conditions and issue the right credentials (badge instances) to those allowed to issue their badges.

**Scenarios:**

- Jane works at a large corporation as a manager and wants to recognise the contribution of a staff member with a “customer relationship level 3” badge. When trying to issue the badge, the issuing platform warns Jane that she doesn’t have the right credentials, and that only those approved by the head of customer services and those who already have a “customer relationship level 4” badge can issue such a badge.

- A professional community has defined 5 levels of proficiency in a competency framework. Each level of competency has its own series of badges and the community has defined that badges at level n should be issued by someone with a similar badge at level n+1 after having been endorsed by at least 10 people, 5 of them within the organisation.

- Citywide Afterschool, a network of education providers offers a set of badges for youth that may be awarded by individual teachers. The organization requires teachers to complete a training, after which they receive an authorization badge that allows them to issue any of Citywide's youth badges. Each of the BadgeClasses in the youth-targeted badge system declare the authorization badge as required for them to be issued by a non-creator profile.

- Citywide Afterschool alternately may wish to award a customized authorization badge that only grants specific privileges for certain of their youth-focused BadgeClasses. They would like to use one authorization badge template that may be customized to apply to specific BadgeClasses.

**Comment by timothyfcook** [comment]
**User Stories**:

1. As an Issuer with many partners, I want to authorize other individuals or organizations to be able to issue a BadgeClass I've defined, so that I can expand the reach of my badges without compromising their credibility.
2. As a creator of a badge who is not also issuing any badges, I want to authorize other individuals or organizations to issue the badges I create.
3. As the creator of a badge, I want to make sure that consumers of badge Assertions  will be able to tell whether the Assertion was issued by an authorized issuer.

**Comment by ottonomy** [comment]
@szerge Tim and I wrote a couple more general versions of these scenarios that don't necessarily prescribe badge awarding as the only possible mechanism to authorizing an issuer. We can investigate if something like a defined authorizedIssuers list would be appropriate as well.

**Comment by 1l2p** [comment]
Delegating authority to issue specific badges has potential to get messy. This seems easier to implement by simply reissuing badges that were issued by one of the bodies I trust to issue "my" badges. So the recipient would receive two badges, one from my trusted partner, and then a second one from me for the exact same evidence / skills / criteria.

**Comment by ottonomy** [comment]
👍 @1l2p. This actually does make more sense as the ability to define requirements by which I as publisher will decide when to accept other issuer's assertions that a badge was awarded. I will cosign/endorse those that meet this criteria.
